.github/workflows/run-checks.yml

```diff
@@ -0,0 +1,44 @@
+name: Run checks
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+.[0-9]+rc[0-9]+"
+
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tox:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        environment:
+          - "py36"
+          - "py37"
+          - "py38"
+
+        include:
+          - environment: "py36"
+            python: "3.6"
+          - environment: "py37"
+            python: "3.7"
+          - environment: "py38"
+            python: "3.8"
+
+    container:
+      image: python:${{ matrix.python }}
+
+    steps:
+      - uses: actions/checkout@master
```

Comment:
it's usually better to use a tagged version of actions to insulate from unreleased and breaking changes

Reply:
Good catch! Thanks! 👍🏻